modules/d/tools/ldc.lua

```diff
@@ -116,6 +116,7 @@
 			SSE3 = "-mattr=+sse3",
 			SSSE3 = "-mattr=+ssse3",
 			["SSE4.1"] = "-mattr=+sse4.1",
+			["SSE4.2"] = "-mattr=+sse4.2",
```

Comment:
Just because I'm not as familiar with D, is SSE 4.2 supported by LDC?

Reply:
I am not able to find documentation for that but at gcc.godbolt.org the LDC compiler accepts it.

Reply:
Good enough for me.